Lib/typing.py

```diff
@@ -1829,7 +1887,8 @@ def __subclasscheck__(cls, other):
                 and cls.__dict__.get("__subclasshook__") is _proto_hook
             ):
                 raise TypeError(
-                    "Protocols with non-method members don't support issubclass()"
+                    f"Protocols with non-method members don't support issubclass()."
+                    f" Non-method members: {cls.__protocol_attrs__}."
```

Comment:
Whoops, forgot to remove this, but should I revert to original or get the actual non-method members?

Reply:
I like this enhancement. Could you separate it out into a separate PR, however? It feels orthogonal to the main purpose of this PR

Reply:
https://github.com/python/cpython/pull/112344